app/utils/importHelpers.ts

```diff
@@ -346,24 +452,36 @@ async function importDuplicateWallets(
         const existingIdentity = existingIdentities[i];
 
         // Find the identity id as it is in the database.
-        const newIdentity = existingData.identities.find((ident) => {
+        const identityInDatabase = existingData.identities.find((ident) => {
             if (
                 ident.identityNumber === existingIdentity.identityNumber &&
-                ident.name === existingIdentity.name &&
                 ident.randomness === existingIdentity.randomness
             ) {
                 return true;
             }
             return false;
         });
 
-        if (!newIdentity) {
+        if (!identityInDatabase) {
             throw new Error(
                 'Internal error. An existing and matching identity should have been found, but was not.'
             );
         }
 
-        const newIdentityId = newIdentity.id;
+        const newIdentityId = identityInDatabase.id;
+
+        // If there is a naming conflict between the names, ask the user to resolve it.
```

Comment:
Does this mean the actual user using the wallet, or isn't this just handled by the resolver function?

Reply:
This is an outdated comment, I have changed it to:
> If there is a name conflict, resolve it.